ios/RNSScreenStack.m

```diff
@@ -83,6 +86,24 @@ - (UIViewController *)reactViewController
 
 - (void)navigationController:(UINavigationController *)navigationController willShowViewController:(UIViewController *)viewController animated:(BOOL)animated
 {
+//  if (animated && [viewController.view isKindOfClass:[RNSScreenView class]] && _controller.transitionCoordinator != nil) {
```

Comment:
Are these left on purpose?

Reply:
These are remainings of the previous implementation which I didn't want to lose. I'll remove them.